tasks/db_deploy/migrations/migrate_versions_3_to_4/migrate.py

```diff
@@ -53,9 +53,6 @@ def migrate_versions_3_to_4(config: Dict[str, str], is_latest_version: bool) ->
         connection.execute(sql.files_table_sql())
         logger.info("files table created.")
 
-        # Commit if there is no issues
-        connection.commit()
```

Comment:
Found by enforcing call-count checks in tests. I'm guessing we don't want to commit unless we can include the schema update.

Reply:
Good catch